lib/immortal.rb

```diff
@@ -57,26 +61,12 @@ def where_with_deleted(conditions)
       end
     end
 
-    def find_with_deleted(*args)
-      ActiveSupport::Deprecation.warn('[immortal] we are deprecating #find_with_deleted use where_with_deleted instead')
-      without_default_scope do
-        find(*args)
-      end
-    end
-
     def where_only_deleted(conditions)
       without_default_scope do
         where(deleted: true).where(conditions)
```

Comment:
O even better, we could add a `mortal` scope that does that for you so we don't add the `where` clause all the time.